internal/router/controllers/system_test.go

```diff
@@ -14,27 +14,115 @@ import (
 func TestSystemControllerMock(t *testing.T) {
 	t.Parallel()
 
-	req, err := http.NewRequest("GET", "/tables/100", nil)
-	require.NoError(t, err)
-
 	systemService := systemimpl.NewSystemMockService()
 	systemController := NewSystemController(systemService)
 
-	router := mux.NewRouter()
-	router.HandleFunc("/tables/{id}", systemController.GetTable)
-
-	rr := httptest.NewRecorder()
-	router.ServeHTTP(rr, req)
-	require.Equal(t, http.StatusOK, rr.Code)
-
-	//nolint
-	expJSON := `{
-		"name":"name-1",
-		"external_url":"https://tableland.network/tables/100",
-		"image":"https://bafkreifhuhrjhzbj4onqgbrmhpysk2mop2jimvdvfut6taiyzt2yqzt43a.ipfs.dweb.link",
-		"attributes":[{"display_type":"date","trait_type":"created","value":1546360800}]
-	}`
-	require.JSONEq(t, expJSON, rr.Body.String())
+	t.Run("get table metadata", func(t *testing.T) {
```

Comment:
As a reminder, the tests here use a mocked implementation. So what is being tested is whether the request and response are as expected.